doc/guides/CustomStartupGuide.md

```diff
@@ -452,8 +455,85 @@ Follow these instructions to update devices.
    ```bash
    [{"apiVersion":"v2","statusCode":201,"id":"fb5fb7f2-768b-4298-a916-d4779523c6b5"}]
    ```
+</details>
+
+<details>
+<summary><strong>Auto-Discovery</strong></summary>  
+
+<br/>
+
+ONVIF devices support WS-Discovery, which is a mechanism that supports probing a network to find ONVIF capable devices.  Refer to [Auto Discovery](../auto-discovery.md) for detailed information on the auto-discovery mechanism.  
+>**NOTE:** Ensure that the cameras are all installed and configured before attempting discovery.    
+
+> For `Netscan`, there is a one line command to determine the `DiscoverySubnets` of your current machine:
```

Comment:
Perhaps, "The following one-line command can be used to discover subnets of your current machine:".